draft-ietf-core-href.md

```diff
@@ -972,18 +992,18 @@ This section provides an analogue to {{Sections 6.4 and 6.5 of -coap}}:
 Computing a set of CoAP options from a request CRI {{decompose-coap}} and computing a
 request CRI from a set of COAP options {{compose-coap}}.
 
-This section makes use of the mapping between CRI scheme ids
+This section makes use of the mapping between CRI scheme numbers
 and URI scheme names shown in {{scheme-map}}:
 
-| CRI scheme id | URI scheme |
-|---------------|------------|
-|            -1 | coap       |
-|            -2 | coaps      |
-|            -7 | coap+tcp   |
-|            -8 | coaps+tcp  |
-|            -9 | coap+ws    |
-|           -10 | coaps+ws   |
-{: #scheme-map title="Mapping CRI scheme ids and URI scheme names"}
+| CRI scheme number | URI scheme name |
```

Comment:
For consistency with the updated names of the registry columns later on, the name of the first column here should be "CRI scheme number value".

Reply:
(See above)